scripts/create-admin-user.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+while [[ -z "$ZULIP_USER_PASS" ]]; do
+    echo "Please enter the Zulip User Domain:"
+    read -s -p ZULIP_USER_DOMAIN
+done
+while [[ -z "$ZULIP_USER_PASS" ]]; do
+    echo "Please enter the Zulip User Full name:"
+    read -s -p ZULIP_USER_FULLNAME
+done
+while [[ -z "$ZULIP_USER_PASS" ]]; do
+    echo "Please enter the Zulip User Email:"
+    read -s -p ZULIP_USER_EMAIL
+done
+while [[ -z "$ZULIP_USER_PASS" ]]; do
+    echo "Please enter the Zulip User Password:"
+    read -s -p ZULIP_USER_PASS
+done
+
+# Doing everything in python, even I never coded in python #YOLO - Alexander T.
+/home/zulip/deployments/current/manage.py shell <<EOF
+from django.core.management.base import CommandError
+from zerver.lib.create_user import create_user
+from zerver.lib.actions import do_change_is_admin
+from zerver.models import Realm, get_realm, email_to_username
+from zerver.decorator import get_user_profile_by_email
+
+try:
+    realm = get_realm('$ZULIP_USER_DOMAIN')
+except Realm.DoesNotExist:
+    raise CommandError("Realm/Domain does not exist.")
+
+try:
+    create_user('$ZULIP_USER_EMAIL', '$ZULIP_USER_PASS', realm, '$ZULIP_USER_FULLNAME', email_to_username('$ZULIP_USER_EMAIL'))
+except:
+    pass
+
+User = get_user_profile_by_email(email='$ZULIP_USER_EMAIL')
+do_change_is_admin(User, True, 'administer')
+User.save()
+quit()
+EOF
```

Comment:
I don't think you need to write a separate script for this; you should be able to do it with:

`./manage.py create_user <args>` followed by `manage.py knight -f user@example.com`

The only reason that wouldn't do what you want is `manage.py create_user` (implemented in zerver/management/commands/create_user.py) doesn't support passing a password argument; can you instead just add such an option?  I'd happily take that as a separate PR which we can quickly merge; no reason to bundle it in here.

Then this code can be just prompting the user and then running those 2 management commands.